src/libsync/bandwidthmanager.cpp

```diff
@@ -85,12 +88,11 @@ BandwidthManager::BandwidthManager(OwncloudPropagator *p) : QObject(),
 
 BandwidthManager::~BandwidthManager()
 {
-    qDebug() << Q_FUNC_INFO;
 }
 
 void BandwidthManager::registerUploadDevice(UploadDevice *p)
 {
-    //qDebug() << Q_FUNC_INFO << p;
+    qCDebug(lcBandwidthManager) << p;
```

Comment:
I kept some, remove others, but that one won't be so useful investigating problems I guess. Removed it.